README.md

```diff
@@ -13,6 +13,8 @@ Here is a checklist of the Data Structures and Algorithms we have or we are plan
 - [x] [Union-Find](./data_structures/union_find/implementation.py)
 - [x] [Graphs](./data_structures/graphs/implementation.py)
 - [ ] Linked List
+- [ ] Disjoint Set
```

Comment:
Two things to notice here:
- We already have the Disjoint Sets data structure added to our list. It is under the name of Union-Find. So it is not necessary to add it.
- Also, it would be great if this PR is only focused on what it says in the title: Adding KMP implementation and tests for that implementation. Maybe add new items to the To-Do list in another PR.

Reply:
ok, my mistake